src/backend/api/reviews.js

```diff
@@ -0,0 +1,71 @@
+const express = require('express');
+const { sum } = require('../database');
+const router = express.Router();
+const knex = require('../database');
+
+// Returns all reviews
+// GET api/reviews/
+router.get('/', async(request, response) => {
+    try {
+        const getAllReviews = await knex('review'); //.select('title');
+        response.json(getAllReviews);
+    } catch (error) {
+        throw error;
+    }
+});
+
+// Adds a new review
+// POST api/reviews/
+router.post('/', async(request, response) => {
+    try {
+        const postReservation = await knex('review').insert({
+            title: request.body.title,
+            description: request.body.description,
+            stars: request.body.stars,
+            created_date: request.body.created_date,
+            meal_id: request.body.meal_id,
+        });
+        response.json(postReservation);
```

Comment:
Most likely you'd want to reply with some simple confirmation message here, rather than leaking what the database replies with.

Reply:
i refactored it